node/impl/eth/transaction.go

```diff
@@ -451,6 +452,10 @@ func (e *ethTransaction) EthGetBlockReceiptsLimited(ctx context.Context, blockPa
 		return nil, xerrors.Errorf("failed to get tipset: %w", err)
 	}
 
+	if limit > api.LookbackNoLimit && ts.Height() < e.chainStore.GetHeaviestTipSet().Height()-abi.ChainEpoch(limit) {
```

Comment:
see the CI error, we now have `make lint` in the Makefile (you may need to rebase, or merge current master in to get it), run that and you should see the lint errors before you push